src/business_logic/fact_state/in_memory_state_reader.rs

```diff
@@ -90,7 +90,7 @@ mod tests {
 
         state_reader
             .ffc
```

Comment:
this field should be renamed since we don't have ffcs

Reply:
Fixed (merge from main to apply the removal of `FactFetchingContext`, then renamed all `ffc`s to `storage`).